repository-data/development/src/main/resources/hcm-content/content/documents/lks/events.yaml

```diff
@@ -262,3 +262,64 @@
       hippostdpubwf:publicationDate: 2021-05-18T12:01:49.739+01:00
       hippotranslation:id: 300dfc60-c232-4d9d-8a91-ef14de917802
       hippotranslation:locale: en
+  /search-event:
```

Comment:
Could we reload `/content/documents/lks/events` ?

Reply:
Added. I've added a few others also.